src/backend/common/MemoryManager.hpp

```diff
@@ -14,6 +14,8 @@
 #include <util.hpp>
 
 #include <algorithm>
+
+// TODO(umar): Remove iostream
 #include <iomanip>
 #include <iostream>
```

Comment:
`iostream` is a pretty big header.  Since this header is going to be included nearly everywhere it is important we keep this header as small as possible.